src/Faker/Provider/Text.php

```diff
@@ -127,6 +128,7 @@ protected static function strlen($text)
 
     protected static function validStart($word)
     {
+        if(  !  $this->textStartsWithUppercase ) return true;
```

Comment:
Stylecheck is already part of the Travis build for Faker (and already detected the problems, see https://travis-ci.org/fzaninotto/Faker/jobs/52732166). I should just write "Read The Fucking Build Fail Result", but I'm being more civil.